### ISSUE TITLE
PostGre DROP FOREIGN KEY = syntax error

### ISSUE THREAD

**Issue by dexit2k** [HeidiSQL/HeidiSQL#1547]
PostgreSQL, libpq-12.dll, HeidiSQL: 12.3.0.6426:
ALTER TABLE "authors_resources" DROP FOREIGN KEY "FK_authors_resources_users";
/* ERROR:  syntax error at or near "FOREIGN" LINE 2:  DROP FOREIGN KEY "FK_authors_resources_users"*/

Working Solution:
ALTER TABLE "authors_resources" DROP CONSTRAINT "FK_authors_resources_users";
/* Betroffene Zeilen: 0  Gefundene Zeilen: 0  Warnungen: 0  Dauer von 1 Abfrage: 0,000 Sek. (+ 0,016 Sek. Netzwerk) */

**Comment by ansgarbecker** [comment]
Yes, the table editor has still quite lots of places with MySQL/MariaDB-only commands. I need to separate these out into snippets for each rdbms.

**Comment by louis77** [comment]
I'm adding myself to this issue, since I have the same problem.

**Comment by ansgarbecker** [comment]
Was solved in #1247 some releases ago.
